Tribler/Main/tribler_main.py

```diff
@@ -950,14 +911,6 @@ def run(params=[""], autoload_discovery=True, use_torrent_search=True, use_chann
         installdir = determine_install_dir()
 
         if not ALLOW_MULTIPLE and process_checker.already_running:
```

Comment:
Users do not look at the log, do they? Shouldn't that broadcast something then which is sent over the evensocket?